src/cart/src/gurt/fault_inject.c

```diff
@@ -669,4 +673,59 @@ d_should_fail(struct d_fault_attr_t *fault_attr)
 	D_SPIN_UNLOCK(&fault_attr->fa_lock);
 	return rc;
 };
+#else
```

Comment:
Helpful for readability to leave comment next to #else  indicating what if condition was e.g. /* FAULT_INJECTION */

Reply:
Added it to both the #else and #endif.